go/vt/vtctl/reparentutil/planned_reparenter.go

```diff
@@ -495,6 +516,8 @@ func (pr *PlannedReparenter) reparentShardLocked(
 	shard string,
 	opts PlannedReparentOptions,
 ) error {
+	prsCounter.Add([]string{keyspace, shard}, 1)
+
```

Comment:
Shouldn't we be incrementing `prsCounter` when we call `ReparentShard` instead of `reparentShardLocked`?

I am thinking that `prsCounter` should be equal to `prsSuccessCounter + prsFailureCounter`.

Reply:
@GuptaManan100 that makes sense. I've updated the new counters to increment in `ReparentShard` 👍

I left the legacy ERS counter in `reparentShardLocked` to avoid changing behaviour for that legacy stat